src/test/java/input_from_screen/CanVideoRecorderPlayWithDiffScreenMetrics.java

```diff
@@ -0,0 +1,79 @@
+package input_from_screen;
+
+import org.junit.Test;
+import tdl.record.screen.image.input.ImageInput;
+import tdl.record.screen.image.input.InputFromScreen;
+import tdl.record.screen.image.input.ScaleToOptimalSizeImage;
+import tdl.record.screen.utils.ImageQualityHint;
+import tdl.record.screen.video.VideoRecorder;
+
+import java.awt.*;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+
+import static junit.framework.TestCase.fail;
+
+public class CanVideoRecorderPlayWithDiffScreenMetrics {
+
+    private static final int EVEN_WIDTH = 300;
+    private static final int ODD_HEIGHT = 151;
+    private static final int ODD_WIDTH = 301;
+    private static final int EVEN_HEIGHT = 150;
+
+    @Test public void
+    given_screen_width_is_odd_adjust_width_and_video_should_still_play_fine() {
+        try {
+            Rectangle screenBounds = new Rectangle(ODD_WIDTH, EVEN_HEIGHT);
+            ImageInput imageInput = new ScaleToOptimalSizeImage(
+                    ImageQualityHint.HIGH,
+                    new InputFromScreen(screenBounds)
```

Comment:
Use InputFromStaticImage instead, as InputFromScreen will not work on the CI (no screen).